STest/STest/Rule/Rule.hpp

```diff
@@ -27,9 +27,9 @@ namespace STest {
 
       //! Construct object Rule
       Rule(
-          const char *const name //!< The name of the rule
+          const char *const a_name //!< The name of the rule
```

Comment:
@bocchino we will need to look into this w.r.t. the STest repository.  If and when we move to a full sub-repo, we'll need to ensure shadows are weeded-out.

I'll leave that to you to track before we make STest a subrepo.